windows/src/engine/keyman/UfrmKeyman7Main.pas

```diff
@@ -305,6 +305,16 @@   TfrmKeyman7Main = class(TForm)
     procedure HotkeyWndProc(var Message: TMessage);
 
     procedure DoLanguageHotkey(Index: Integer);
+
+    // #5004 Investigation Start
+    procedure CMRecreateWnd(var Message: TMessage); message CM_RECREATEWND;
+  private
+    FRecreateWndCalled: Boolean;
+  protected
+    procedure CreateWindowHandle(const Params: TCreateParams); override;
+    procedure DestroyWindowHandle; override;
```

Comment:
Is there a separate PR where these new logging procedures get called?

Reply:
Those are `override` functions, called by the Delphi VCL.